lib/components/Button/Button.tsx

```diff
@@ -63,66 +64,72 @@ const progressSpinnerVariantMap: Map<
 ]);
 
 // TODO: Fix this any here, needs to be an abstract of the component that is being passed in.
-export const Button: FunctionComponent<IProps & any> = ({
-	children,
-	className = '',
-	is: Component = 'button',
-	component = void 0,
-	disabled = false,
-	isLoading = false,
-	isFullWidth = false,
-	rounded = false,
-	size = ESize.Medium,
-	variant = EVariant.Secondary,
-	...rest
-}) => {
-	// @deprecated block
-	{
-		warning(
-			component !== void 0,
-			`The \`component\` prop deprecated, please use the \`is\` prop instead.\n\nBefore:\n<Button component="{<Link/">}>\n\tHello\n</Button>\n\nAfter:\n<Button is="{<Link/">}>\n\tHello\n</Button>`
-		);
+export const Button: FunctionComponent<IProps & any> = forwardRef(
```

Comment:
Should we consider doing same thing for other similar components lie Anchor for example?